Azure/azure-cli
# [App Service] `az functionapp deployment source config-zip`: Adding the deployer information to improve telemetry

**Related command**
`az functionapp deployment source config-zip`

**Description**
Adding the deployer information enables us to improve telemetry.

**Testing Guide**
1. Create a function app with the `az functonapp create` command
2. Deploy code with the `az functionapp deployment source config-zip` command

---

This checklist is used to make sure that common guidelines for a pull request are followed.
